compiler-rt/lib/builtins/muldf3.c

```diff
@@ -14,7 +14,8 @@
 #define DOUBLE_PRECISION
 #include "fp_mul_impl.inc"
 
-COMPILER_RT_ABI fp_t __muldf3(fp_t a, fp_t b) { return __mulXf3__(a, b); }
+COMPILER_RT_ABI fp_t
```

Comment:
whitespace change?

Reply:
Yes, because of 7dccccfa followed by 94a33d22. Will rollback that whitespace-only change.